nvtabular/inference/triton/__init__.py

```diff
@@ -248,7 +328,14 @@ def convert_triton_output_to_df(columns, response):
 
 
 def _generate_nvtabular_config(
```

Comment:
I'm wondering if we should split this function up into 3 (for tf/hugectr/pytorch). There isn't much code in common between the 3 options

Reply:
lets track in https://github.com/NVIDIA/NVTabular/issues/708